src/pages/permissions/actions/BulkDeleteIdentitiesBtn.tsx

```diff
@@ -0,0 +1,83 @@
+import { FC } from "react";
+import {
+  ButtonProps,
+  ConfirmationButton,
+  useNotify,
+} from "@canonical/react-components";
+import { LxdIdentity } from "types/permissions";
+import { deleteOIDCIdentities } from "api/auth-identities";
+import { useQueryClient } from "@tanstack/react-query";
+import { useToastNotification } from "context/toastNotificationProvider";
+import { queryKeys } from "util/queryKeys";
+
+interface Props {
+  identities: LxdIdentity[];
+  className?: string;
+}
+
+const BulkDeleteIdentitiesBtn: FC<Props & ButtonProps> = ({
+  identities,
+  className,
+}) => {
+  const queryClient = useQueryClient();
+  const notify = useNotify();
+  const toastNotify = useToastNotification();
+  const buttonText =
+    identities.length > 1
+      ? `Delete ${identities.length} identities`
+      : "Delete identity";
+  const successMessage =
+    identities.length > 1
+      ? "Identities successfully deleted"
+      : "Identity deleted";
```

Comment:
I think we can use our own `pluralize` here, to make this a bit simpler, and both will become one-liners.